Engines/Wine/Verbs/DXVK/script.js

```diff
@@ -58,3 +59,35 @@ Wine.prototype.DXVK = function (dxvkVersion) {
 
     return this;
 }
+
+/**
+ * Verb to install DXVK
+*/
+var verbImplementation = {
```

Comment:
I would like some comments or paragraphs in this verb implementation which describe/section what happens when

Reply:
I don't quite understand. It shows the wizard to select the version and then installs the selected version. What comments would you like to have

Reply:
I would wish for something like:
```javascript 
/**
 * 1. download latest dxvk
 * 2. select the to be installed version
 **/
```

In addition I must confess that I have no clue about what version we're talking, because the download already happened?

Reply:
Done.